### PR TITLE
Indirect structs

### PR DESCRIPTION
**Pull request by Przemog1**

### CODE REVIEW

File: include/irr/asset/IMeshBuffer.h [hunk 1]
```diff
@@ -42,6 +42,13 @@ struct DrawElementsIndirectCommand_t
     uint32_t baseInstance;
 };
 
+struct DispatchIndirectCommand_t
```
Comment (Crisspl) [inline include/irr/asset/IMeshBuffer.h]: I'd move this to IComputePipeline.h or make separate utility header and move all 3 there

Comment (Crisspl) [reply include/irr/asset/IMeshBuffer.h]: Probably separate header is best option

Comment (Przemog1) [reply include/irr/asset/IMeshBuffer.h]: @devshgraphicsprogramming what do you think?

Comment (devshgraphicsprogramming) [reply include/irr/asset/IMeshBuffer.h]: I second criss, this (all indirect structs) should move to `irr/asset/IPipeline.h`

**Pull request merged**